profiles/models.py

```diff
@@ -142,6 +180,8 @@ class Profile(models.Model):
     linkedin = JSONField(blank=True, null=True)
     student_id = models.IntegerField(blank=True, null=True, unique=True)
 
+    date_updated = models.DateTimeField(blank=True, null=True, auto_now=True)
```

Comment:
@gsidebo Hm, on further inspection, `TimestampedModel` doesn't declare it's fields as nullable which is problematic for a table that already has records. There also is already a `date_joined_micromasters` which is `auto_now_add` so adding `created_on` that you'd get from `TimestampedModel` would be repetitive. The good news there is that we would have values for `created_on` for existing records should we wish to switch to that.

I'd suggest as an intermediate step that I switch the definition of this new field to match `created_on` in `TimestampedModel` and then file a new issue that we should normalize `Profile` to `TimestampedModel` since that's nontrivial and should be tackled separately from the work I'm doing here.

Thoughts?